controllers/config.go

```diff
@@ -0,0 +1,57 @@
+package controllers
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	configv1 "github.com/openshift/api/config/v1"
+)
+
+// imagesReference allows build systems to inject imagesReference for CCCMO components
+type imagesReference struct {
+	CloudControllerManagerAWS       string `json:"cloudControllerManagerAWS"`
+	CloudControllerManagerOpenStack string `json:"cloudControllerManagerOpenStack"`
+}
```

Comment:
i'm just curious, will a cluster ever use multiple images from this reference?

Reply:
Yes, this is possible, there is definitely going to be a `kube-rbac-proxy` image here once we will start investigating metrics support.